### PR TITLE
[2/N] Fix clang-tidy warnings in torch/csrc/jit/serialization

### PR DESCRIPTION
**Pull request by cyyever**
Follows #129055

### CODE REVIEW

**Comment by Skylion007** [comment]
Contains is a C++20 feature, isn't it?

**Comment by cyyever** [comment]
@Skylion007 C++20 somehow sneaked in, now it is fixed.

File: torch/csrc/jit/serialization/export.cpp [hunk 1]
```diff
@@ -1078,6 +1080,7 @@ void GraphEncoder::AddAttribute(
       ATenAttributeKindToOnnxAttributeType(node->kindOf(name), name));
   switch (node->kindOf(name)) {
     case AttributeKind::f:
+      // NOLINTNEXTLINE(bugprone-narrowing-conversions,cppcoreguidelines-narrowing-conversions)
```
Comment (Skylion007) [inline torch/csrc/jit/serialization/export.cpp:1083]: We can't fix these with explicit casts?

Comment (cyyever) [reply torch/csrc/jit/serialization/export.cpp]: Fixed

**Review by ezyang — APPROVED** [review]

**Pull request closed without merging**